packages/rmb_direct_client/lib/client.ts

```diff
@@ -167,9 +167,27 @@ class Client {
     }
   }
 
-  disconnect() {
+  private async waitForResponses(): Promise<void> {
+    const start = new Date().getTime();
+    while (new Date().getTime() < start + 2 * 60 * 1000) {
```

Comment:
Making it a parameter is beneficial so that it can be easily adjusted if needed.
```suggestion
private async waitForResponses(timeoutInMilliseconds: number = 2 * 60 * 1000): Promise<void> {
    const start = new Date().getTime();
    while (new Date().getTime() < start + timeoutInMilliseconds) {
```